.github/workflows/release-charts.yaml

```diff
@@ -0,0 +1,104 @@
+      pull-requests: write
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      HELM_SIGNING_KEY_ID: ${{ secrets.HELM_SIGNING_KEY_ID }}
+      GITHUB_REPOSITORY: ${GITHUB_REPOSITORY}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.9.4
+
+      - name: Setup Python environment
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9.14'
+
+      - name: Install Python dependencies
+        run: |
+          python -m pip install --upgrade pip gitpython ruamel_yaml
+
+      - name: Generate Release Notes for GitHub Releases
+        run: |
+          cd src/main/scripts
+          products=(bamboo bamboo-agent bitbucket confluence crowd jira)
+          export REPO_PATH="${GITHUB_WORKSPACE}"
+          export CHARTS_LOCATION="src/main/charts"
+          for product in "${products[@]}"; do
+            echo "[INFO]: Generating ${REPO_PATH}/${CHARTS_LOCATION}/${product}/RELEASE_NOTES.md"
+            REL_NOTES_LIST=$(python3 -W ignore -c "import prepare_release; print (prepare_release.gen_changelog(\"${product}\", \"${REPO_PATH}\"))")
+            python -c "list=${REL_NOTES_LIST}; print('\n'.join(list))" > "${REPO_PATH}"/"${CHARTS_LOCATION}"/"${product}"/RELEASE_NOTES.md
+          done
+
+      - name: Execute helm dependency update for bamboo chart
+        run: helm dependency update src/main/charts/bamboo
+      - name: Execute helm dependency update for bamboo-agent chart
+        run: helm dependency update src/main/charts/bamboo-agent
+      - name: Execute helm dependency update for bitbucket chart
+        run: helm dependency update src/main/charts/bitbucket
+      - name: Execute helm dependency update for confluence chart
+        run: helm dependency update src/main/charts/confluence
+      - name: Execute helm dependency update for crowd chart
+        run: helm dependency update src/main/charts/crowd
+      - name: Execute helm dependency update for jira chart
+        run: helm dependency update src/main/charts/jira
+
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@master
+        with:
+          aws-region: ${{ secrets.HELM_SIGNING_KEY_AWS_REGION }}
+          role-to-assume: ${{ secrets.HELM_SIGNING_KEY_AWS_IAM_ROLE }}
+          role-session-name: HelmSignSession
+
+      - name: Get Helm Signing Key
+        run: |
+          aws secretsmanager get-secret-value --secret-id ${{ secrets.HELM_SIGNING_KEY_AWS_SECRET_ID }} --query 'SecretBinary' --output text --region ${{ secrets.HELM_SIGNING_KEY_AWS_REGION }} | base64 --decode > helm_key
+          gpg --import helm_key
+          gpg --export > ~/.gnupg/pubring.gpg
```

Comment:
We might not need to export pub key ring to the legacy format here. As it's already exported on the second next line below.

Reply:
Right, this export is needed if we need to verify the package. I think I left it for a possible future test case where we may want to run helm verify after signing (not implemented yet)